### DISCUSSION TITLE
Get resource list 500. Unable to retrieve popular movies.

### DISCUSSION THREAD

**Discussion by myronjava** [seerr-team/seerr#1077] [Q&A] [answered]
Viewing the log shows that: [debug][API]: Something went wrong retrieving popular movies {"errorMessage":"[TMDB] Failed to fetch discover movies: fetch failed"}

Originally is the normal access to film and television resources list, suddenly no longer, the server network is normal, can not find more detailed logs, May I ask is the external dependency service out of the problem or jellyseerr problem

**Comment by myronjava** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/283a5c37-b004-4aa1-8782-8e14b90428f0)

The service runs in docker, and other network resources requested by radarr,sonarr and jackett are all normal. This problem occurs in jellyseerr. May I ask whether it is a service problem or a container network problem or an external dependent service problem? These services all run in the same docker

**Reply by fallenbagel** [reply]
This is an internal server error. To troubleshoot please hop on discord

**Comment by Romzes25** [comment]
у меня таже проблема. ни где нету информации об этом , discord не доступен

**Reply by fallenbagel** [reply]
The issue for you was addressed in the other discussion you commented in. Your issue probably is tmdb is blocked in your country. Your solution is the use of a vpn/http proxy, as advised on the other discussion 

https://github.com/Fallenbagel/jellyseerr/discussions/959#discussioncomment-11445595
